app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,39 @@
+
+html, body {
+  background: #FAFAFA; }
+
+.card {
```

Comment:
if you have something named 'Card' in your layout you can go to assets/stylesheets/modules/ and place there new file _card.scss and keep card's style in it. do not forget to require new file in all.scss